src/java/fr/paris/lutece/portal/service/editor/EditorBbcodeService.java

```diff
@@ -91,14 +91,13 @@ public static EditorBbcodeService getInstance( )
     {
         if ( _singleton == null )
         {
-            synchronized( EditorBbcodeService.class )
+            synchronized ( EditorBbcodeService.class )

```

Comment:
same remarks here

Reply:
Corrected in another commit of the same PR.